### ISSUE TITLE
Wireshark dissector update?

### ISSUE THREAD

**Issue by damienrg** [openambitproject/openambit#173]
Hi,
I would like to update the wireshark dissector in order to:
 - make it working with newer version of wireshark
 - make it working when compiling from wireshark source or as is (ie without wireshark source)
 - maybe send this dissector upstream
To begin implementing this I have the following questions:
 - should I be backward compatible with the previous wireshark version?
 - do you prefer that I start a new repository including only the wireshark dissector? I think that it is better if we want to send this upstream as I could have 2 branch one for upstream and one for openambit.

**Comment by manisandro** [comment]
I know nothing about wireshark, but is there really the need to keep two completely separate codebases/branches for wireshark upstream and openambit?

**Comment by damienrg** [comment]
For wireshark upstream, we need:
  - different CMakeLists.txt as it is included in the wireshark code
  - Makefile.am, plugin.rc.in
Of course, we will always keep the same code for the dissector, it is just that the plumbing are different.

**Comment by manisandro** [comment]
So if the plugin is successfully sent upstream, I take the respective plumbing will just live upstream? Or is there a need to keep a downstream copy?

**Comment by damienrg** [comment]
It depends on how you build the plugin but I think that you build that out of the wireshark source. So in this case there is no need to keep the plumbing here.
When openambit folks release a new version of the plugin someone should report it upstream to make it included in wireshark. To not bother openambit folks I can send PR to modify the plugin here, do the plumbing in my repo and sent it upstream.

**Comment by manisandro** [comment]
Makes sense to me. About backward compatibility: I'd say if you can keep compatibility with the wireshark versions current distros are packaging, it would simplify life for the packagers.
